docs/en/development/CI.md

```diff
@@ -0,0 +1,209 @@
+## CI checks
+
+When you submit a pull request, some automated checks are ran for your code by
```

Comment:
line breaks will be rendered in html as well, so paragraphs are usually represented as a single line in markdown

Reply:
It's just for my editor, may be removed.

Reply:
> line breaks will be rendered in html as well, so paragraphs are usually represented as a single line in markdown

This is just not true, neither in [GitHub flavor](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#paragraphs), nor in [Commonmark](https://spec.commonmark.org/0.29/#paragraphs), nor in the [original spec](https://daringfireball.net/projects/markdown/syntax#p). Paragraphs can contain any number of lines and are delimited by a single empty line. If our docs rendered doesn't respect that, it's broken.

Reply:
+1. Markdown processing in our documentation is just broken and should be fixed.